src/components/home/MessageBox.js

```diff
@@ -18,14 +19,28 @@ export default class MessageBox extends Component {
       }
   }
 
+  dropMessage() {
+      //figure out json structure of each "drop" object
+      // object needs:
+      //    unique id
+      //    user info( need to know who to attribute to) (can set to anon)
+      //    location (long/lat)
+      //    radius it will be visible 
+      //    message
+      //    img?
+      //    life span
+      //    store to firebase db
```

Comment:
this is just the object properties that you think we'll need, designed by you, right?  img could be int he future I imagine?
also, is this already designed?

Reply:
by designed i mean, written in code somewhere

Reply:
No this is not made yet, I wanted your input in terms of how you think this object shoudl be structured and what other data should be included